### DISCUSSION TITLE
How to use a private image repo other than ghcr.io while triggering a "Run" or "Deploy" in Karavan

### DISCUSSION THREAD

**Discussion by kasiviswanathan13** [apache/camel-karavan#1409] [General]
My org prevents pulling images directly from the internet. So for deploying Karavan, we pulled the image into our private jFrog and then used it for deployment of Karavan. When we were trying to create antest integration and "Run" or "Deploy' it we got an error saying access to ghcr.io is blocked. How can i customize this? LIke is it possible to edit the builder.pod.jkube.yaml and get this working?

**Comment by mgubaidullin** [comment]
1. Pull ghcr.io/apache/camel-karavan-devmode:4.7.0 and push it to your private image registry
2. Set KARAVAN_DEVMODE_IMAGE environment variable to your image
